README.md

```diff
@@ -120,7 +120,7 @@ available configuration parameters, see the [included example][5]).
     NCP:CCAThreshold              "-70"
 
 When up and running, `wpanctl` can be used to check the status of the
-interface and perform various management operations. For example, to
+interface and perform various management operations, for example, to
```

Comment:
I personally feel like this reads better with a hard stop, or at least a semicolon.